prisma/schema/campaignTask.prisma

```diff
@@ -0,0 +1,33 @@
+enum CampaignTaskType {
+    text
+    robocall
+    doorKnocking
+    phoneBanking
+    socialMedia
+    events
+    education
+}
+
+model CampaignTask {
+    id                  Int              @id @default(autoincrement())
+    createdAt           DateTime         @default(now()) @map("created_at")
+    updatedAt           DateTime         @updatedAt @map("updated_at")
+    taskId              String           @unique @map("task_id")
```

Comment:
Why is there a `taskId` _and_ an `id`?

We should just have an `id` and should be a UUID, not an `Int`.. why regress back to `Int` when UUID was working fine?